src/google_cloud_debugger/google_cloud_debugger_lib/dbg_class_field.cc

```diff
@@ -170,6 +179,100 @@ HRESULT DbgClassField::Evaluate(
   return S_OK;
 }
 
+HRESULT DbgClassField::ProcessConstField(ICorDebugModule *debug_module,
+                                         IMetaDataImport *metadata_import) {
+  CorElementType field_type = (CorElementType)default_value_type_flags_;
+  ULONG64 enum_numerical_value = 0;
+  std::unique_ptr<DbgObject> dbg_object;
+  HRESULT hr = obj_factory_->CreateDbgObjectFromLiteralConst(
+      field_type, default_value_, default_value_len_, &enum_numerical_value,
+      &dbg_object);
+
+  if (FAILED(hr)) {
+    return hr;
+  }
+
+  // If this is an enum, the signature length will be greater than 2
+  // and the second byte will be CorElementType::ELEMENT_TYPE_VALUETYPE.
```

Comment:
super nit: I generally think it's easier to do the equals case first.  This comment makes me think this would be the next logical part.  Just a suggestion.

```
  if (second_byte_type == CorElementType::ELEMENT_TYPE_VALUETYPE) {
    return ProcessConstEnumField(debug_module, metadata_import, field_type, enum_numerical_value);
  }
  member_value_ = std::move(dbg_object);
   return S_OK; 
```